src/lib/containers/user_profile_links/UserChallenges.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react'
+import { useErrorHandler } from 'react-error-boundary'
+import { toError } from '../../utils/ErrorUtils'
+import { useInView } from 'react-intersection-observer'
+import { SynapseSpinner } from '../LoadingScreen'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../../utils/functions/getEndpoint'
+import { useGetUserChallengesInfinite } from '../../utils/hooks/SynapseAPI/useGetUserChallenges'
+import { ChallengeWithProjectHeader } from '../../utils/synapseTypes/ChallengePagedResults'
+
+export type UserChallengesProps = {
+  userId: string
+}
+
+export default function UserChallenges({userId}:UserChallengesProps) {
+  const handleError = useErrorHandler()
```

Comment:
Do these components have an ancestor ErrorBoundary component? I didn't see one but I'm not sure.

Reply:
Good call, will add a SynapseErrorBoundary to the UserProfileLinks parent component